airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/format/check/commands.py

```diff
@@ -0,0 +1,100 @@
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+
+import logging
+import sys
+from typing import List
+
+import anyio
+import asyncclick as click
+import dagger
+from pipelines.airbyte_ci.format.actions import run_check
+from pipelines.airbyte_ci.format.check.utils import log_output
+from pipelines.airbyte_ci.format.containers import (
+    format_java_container,
+    format_js_container,
+    format_license_container,
+    format_python_container,
+)
+from pipelines.cli.click_decorators import click_ignore_unused_kwargs, click_merge_args_into_context_obj
+from pipelines.models.contexts.click_pipeline_context import ClickPipelineContext, pass_pipeline_context
+
+
+@click.group(
+    help="Run code format checks and fail if any checks fail.",
+    invoke_without_command=True,
+    chain=True,
+)
+@click.option("--list-errors", is_flag=True, default=False, help="Show detailed error messages for failed checks.")
+@click_merge_args_into_context_obj
+@pass_pipeline_context
+@click_ignore_unused_kwargs
+async def check(ctx: click.Context, pipeline_ctx: ClickPipelineContext, list_errors: bool):
+    """Run code format checks and fail if any checks fail."""
+    logger = logging.getLogger("check")
+
+    ctx.obj["dagger_client"] = await pipeline_ctx.get_dagger_client(pipeline_name="Check repository formatting")
```

Comment:
@bnchrch I don't think I can safely attach/store this one in the ClickPipelineContext, but I'm hoping you can prove me wrong? 

I dig it back out for use [here](https://github.com/airbytehq/airbyte/pull/31831/files#diff-965ce6ff9499e8fecfc40eab13db5482e458a201ab7244588a3713062ebb7baeR33)

Reply:
Looking at this it seems like we are stashing it away so that we can share a pipeline name?